astropy/convolution/tests/test_pickle.py

```diff
@@ -1,8 +1,11 @@
 # Licensed under a 3-clause BSD style license - see LICENSE.rst
 from __future__ import absolute_import, division, print_function, unicode_literals
+
+import pytest
 import numpy as np
+
 from ... import convolution as conv
-from ...tests.helper import pytest, pickle_protocol, check_pickling_recovery
+from ...tests.helper import pickle_protocol, check_pickling_recovery  # noqa
```

Comment:
Same question about `noqa`, except that here at least one test does get run, so why is it needed?